code/modules/research/designs/autolathe_designs.dm

```diff
@@ -435,6 +435,8 @@
 	category = list("initial", "Medical", "Tool Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL | DEPARTMENTAL_FLAG_SCIENCE
 
+// Wondering where the surgical_drapes are? See yogs medical_designs, because the Autolathe does not support MAT_PLASTIC yet!
```

Comment:
Why?

Reply:
I'll remove it, I figured the comment would help people find the surgical drapes because that's where the rest of the surgical tool designs are and where coders might go looking for them

Reply:
Fixed. :)